backends/arm/runtime/ArmBackendEthosU.cpp

```diff
@@ -15,6 +15,17 @@
 
 #include <ethosu_driver.h>
 
+#if defined(ET_EVENT_TRACER_ENABLED)
+#include <executorch/runtime/core/event_tracer.h>
+#include <executorch/runtime/core/event_tracer_hooks.h>
+using executorch::runtime::internal::EventTracerProfileOpScope;
+#define EXECUTORCH_INTERNAL_PROF(NAME)              \
+  EventTracerProfileOpScope event_tracer_op_scope = \
+      EventTracerProfileOpScope(event_tracer, NAME);
```

Comment:
Yeah AFAICT this is not meant to be used from outside, as you were suspecting.

So I would try using stuff from `executorch/runtime/core/event_tracer.h` like `EventTracer::[start|stop]_profiling`?

Reply:
Yes that works, PR will be updated soon.

Reply:
Fixed!